orangecontrib/imageanalytics/widgets/owimagenetembedding.py

```diff
@@ -33,6 +35,7 @@ class OWImageNetEmbedding(widget.OWWidget):
                ("Missing Images", Table)]
     auto_commit = Setting(True)
     token = Setting("")
+    img_attr = ContextSetting(0)
```

Comment:
Context settings need a corresponding `settingsHandler` definition (in this case probably a `DomainContextHandler`), and later in the `set_data` an {close,open}Context call pair .